api/all/src/main/java/io/opentelemetry/api/baggage/ImmutableBaggage.java

```diff
@@ -94,7 +94,7 @@ public Baggage build() {
    * @return whether the name is valid.
    */
   private static boolean isKeyValid(String name) {
-    return name != null && !name.isEmpty() && StringUtils.isPrintableString(name);
+    return name != null && !name.trim().isEmpty() && StringUtils.isPrintableString(name);
```

Comment:
can you add a unit test for this? Thanks!

Reply:
@jkwatson 

I added a unit test for it, but not sure if it's OK (it's my first collaboration on this project). Please tell me if I should do anything else. Thanks!

Reply:
this is an unnecessary allocation with "trim". Can we avoid that?

Reply:
If "   " is invalid, then "  t" means is equal with "t" ?

Reply:
I think we will adopt the w3c standard and will be https://w3c.github.io/baggage/#key